exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -198,6 +198,7 @@ def __init__(
 
         if parsed_url.netloc:
             endpoint = parsed_url.netloc
+        self._endpoint = endpoint
```

Comment:
Kind of a nit, but I wonder if it would be a bit clearer to treat `endpoint` as the incoming arg and not overwrite it (it's first overwritten on line 181), and to treat `self._endpoint` as the resolved value (to be used to pass to other functions, etc.). Then you wouldn't have to add this line.

Reply:
Thanks for the review @pmcollins. I applied your suggestion. PTAL